@navikt/core/react/src/table/stories/table-async.stories.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from "react";
+import { rest } from "msw";
+import useSWR from "swr";
+import people from "./people.json";
+import { Table } from "../index";
+import { Loader, Pagination, SortState } from "../..";
+
+export default {
+  title: "ds-react/table",
+  component: Table,
```

Comment:
Passe på at data er lastet før snapshot
```suggestion
  component: Table,
  parameters: {
    chromatic: { delay: 500 },
  },
```